platform/understand/externally-vs-platform-deployed.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Virtual Cluster Lifecycle
+sidebar_label: Virtual Cluster Lifecycle
+---
+
+# Virtual Cluster Lifecycle
+
+There are two main types of virtual clusters based on how they're deployed: platform-deployed and externally deployed. These classifications determine how the clusters are created, updated, and integrated with the platform's features. Understanding the differences between these types is important for effective cluster management and utilization of platform capabilities.
+
+## Platform-Deployed Virtual Clusters
+
+These are virtual clusters that are deployed and fully managed through our
+platform, regardless if they are deployed via the UI or CLI. They benefit from the platform's complete set of management features and automated lifecycle management.
+
+:::info Characteristics
+- Deployment methods:
+    - Platform UI
+    - vCluster CLI (when authenticated to Platform)
+- Full platform lifecycle management and reconciliation
+- Complete access to platform-specific features
+:::
+
+Platform-deployed clusters provide an integrated experience with centralized management and full platform feature utilization.
+
+## Externally Deployed Virtual Clusters
+
+These virtual clusters are initially deployed outside of our platform, using tools like Helm or custom scripts. While they can be imported into the platform for some level of management, they retain their original deployment method and have limited platform integration.
+
+:::info Characteristics
+- Deployed using external tools (e.g., ArgoCD, CAPI)
+- Deployed using `vcluster create <name>` command (if driver helm is active)
+- Deployed using `helm install vcluster ...`
+- Added to the platform using `vcluster platform add vcluster` command
+- Platform won't manage it (perform reconciliation)
+:::
+
+In this scenario, changes to externally deployed virtual clusters should not be performed via the Platform UI. Instead, they should be made using the same tool originally used to deploy the virtual cluster (for example ArgoCD, ClusterAPI, or other tools).
+
+## Understanding the Differences
+
+Virtual clusters are classified based on two key factors: their creation method (via the platform or externally) and their update mechanism (managed by the platform or an external tool). These factors determine whether the platform or an external tool is the deployment owner.
+
+:::note
+In short, the key differences are:
+- Creation method: Platform interface vs. external tools
+- Update mechanism: Platform-driven vs. externally controlled
+:::
+
+📝 Current limitations for externally deployed clusters:
+- Sleep mode unavailable
```

Comment:
This is inaccurate. 

Sleep mode is available if you connect the cluster to the platform. 

We need to differentiate that even though these are externally deployed, there are some features that can only be used if the agent is installed. For example, sleep mode. 

Specifically any feature in the 
```
platform:
```

section of the vcluster.yaml is only available if the agent is installed.

Reply:
Updated